main.go

```diff
@@ -232,6 +232,7 @@ func initializeManager(config *managerConfig) (manager.Manager, error) {
 
 func main() {
 	logger := setupLogger()
+	_ = setupLogger()
```

Comment:
whats the purpose of this change? could you please elaborate?

Reply:
just approve. we're testing an approval process